packages/core/src/Table/stories/TableSection/TableSettings.tsx

```diff
@@ -0,0 +1,522 @@
+            setFixedColumns?.((p) => {
+              if (!p.includes(ts[activeIndex].id)) {
+                return [...p, ts[activeIndex].id];
+              }
+              return p;
+            });
+          } else {
+            setFixedColumns?.((p) => {
+              const newFixed = p.filter((id) => id !== ts[activeIndex].id);
+              return newFixed;
+            });
+          }
+        }
+        return newArray;
+      });
+    }
+
+    const isOverAColumn = over.data.current?.type === "Column";
+
+    // Dropping a Task over a Column
+    if (isActiveATask && isOverAColumn) {
+      setTasks((ts) => {
+        const activeIndex = ts.findIndex((t) => t.id === activeId);
+
+        ts[activeIndex].columnId = overId.toString();
+
+        if (overId.toString() === "hidden") {
+          onColumnHide?.(ts[activeIndex].id, "hide");
+          setFixedColumns?.((p) => {
+            const newFixed = p.filter((id) => id !== ts[activeIndex].id);
+            return newFixed;
+          });
+        } else {
+          onColumnHide?.(ts[activeIndex].id, "show");
+          if (overId.toString() === "fixed") {
+            setFixedColumns?.((p) => {
+              const uniqueIds = new Set([...p, ts[activeIndex].id]);
+              return Array.from(uniqueIds);
+            });
+          } else {
+            setFixedColumns?.((p) => {
+              const newFixed = p.filter((id) => id !== ts[activeIndex].id);
+              return newFixed;
+            });
+          }
+        }
+        return arrayMove(ts, activeIndex, activeIndex);
+      });
+    }
+  };
+
+  return (
+    <DndContext
+      sensors={sensors}
+      onDragStart={onDragStart}
+      onDragEnd={onDragEnd}
+      onDragOver={onDragOver}
+    >
+      <SortableContext items={columnsId}>
+        {columns.map((column) => (
+          <ColumnContainer
+            key={column.id}
+            column={column}
+            tasks={tasks.filter((task) => task.columnId === column.id)}
+            total={tasks.length}
+          />
+        ))}
+      </SortableContext>
+      <DragOverlay
+        modifiers={[
+          restrictToWindowEdges,
+          (args) => myModifier(bdId || "", args),
+        ]}
+      >
+        {activeColumn && (
+          <ColumnContainer
+            column={activeColumn}
+            tasks={tasks.filter((task) => task.columnId === activeColumn.id)}
+          />
+        )}
+        {activeTask && <TaskCard task={activeTask} />}
+      </DragOverlay>
+    </DndContext>
+  );
+};
+
+export const TableSettings = () => {
+  const tableColumns = useMemo(() => getColumns(), []);
+  const [popperOpen, setPopperOpen] = useState(false);
+  const [data] = useState(makeData(5));
+  const [columnOrder, setColumnOrder] = useState(tableColumns.map((c) => c.id));
+  const [hiddenColumns, setHiddenColumns] = useState<string[]>([]);
+  const [fixedColumns, setFixedColumns] = useState<string[]>([]);
```

Comment:
If we switch the order between the fixed columns, sometimes the border is not on the right column:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/9430f756-911b-4897-9a81-9b735bcbd22f